compose/.env.example

```diff
@@ -6,7 +6,7 @@
 # Please refer to our documentation to see all possible variables:
 #   https://www.openproject.org/docs/installation-and-operations/configuration/environment/
 #
-TAG=12
+TAG=13
```

Comment:
![image](https://github.com/opf/openproject-deploy/assets/6209340/f183bfad-ffa4-414f-afc8-afeba790a80f)

Latest as today is 13. Also I tested this yesterday when I deployed a new instance
[13.0.7](https://hub.docker.com/layers/openproject/community/13.0.7/images/sha256-5bb33bcdfe253cd8e5d8d5006f09421753ba61ad3ebc6eb20763f4b791fdf41f?context=explore)